src/hooks/useCart/useCart.test.tsx

```diff
@@ -297,6 +306,58 @@ describe("useCart", () => {
       );
     });
 
+    it("should throw cartError (invalidQuantity) when quota response has empty quantities", () => {
+      expect.assertions(1);
+      const ids = ["ID1"];
+      const { result } = renderHook(
+        () =>
+          useCart(ids, key, endpoint, mockQuotaResEmptyQuota.remainingQuota),
+        {
+          wrapper,
+        }
+      );
+      expect(result.current.cartError?.message).toBe(
+        ERROR_MESSAGE.INVALID_QUANTITY
+      );
+    });
+
+    it("should throw cartError (missingDisbursements) when quota response has empty quantities", () => {
+      expect.assertions(1);
+      const MissingDisbursementsWrapper: FunctionComponent = ({ children }) => (
+        <CampaignConfigsStoreContextProvider>
+          <CampaignConfigContextProvider
+            campaignConfig={{
+              features: {
+                ...defaultFeatures,
+                apiVersion: "v2",
+                campaignName: "Some Campaign Name",
+              },
+              policies: [defaultProducts[0]],
+              c13n: {},
+            }}
+          >
+            <ProductContextProvider products={defaultProducts}>
```

Comment:
Since this is the same as `wrapper`, should this be replaced with `{ wrapper }`?

Reply:
hmm i tried this but it doesnt seem to work, not sure if it's possible to perform abstraction here tho! any idea?